ibm_catalog.json

```diff
@@ -0,0 +1,574 @@
+              "required": true
+            },
+            {
+              "key": "existing_subnet_id",
+              "required": true,
+              "custom_config": {
+                "type": "platform_resource",
+                "grouping": "deployment",
+                "original_grouping": "deployment",
+                "config_constraints": {
+                  "resourceType": "subnet",
+                  "selection": "single_select",
+                  "valueType": "id"
+                },
+                "associations": {
+                  "parameters": [
+                    {
+                      "name": "region",
+                      "optionsRefresh": true
+                    }
+                  ]
+                }
+              }
+            },
+            {
+              "key": "image_id",
+              "required": true
+            },
+            {
+              "key": "provider_visibility",
+              "options": [
+                {
+                  "displayname": "private",
+                  "value": "private"
+                },
+                {
+                  "displayname": "public",
+                  "value": "public"
+                },
+                {
+                  "displayname": "public-and-private",
+                  "value": "public-and-private"
+                }
+              ]
+            },
+            {
+              "key": "vsi_name"
+            },
+            {
+              "key": "vsi_resource_tags",
+              "custom_config": {
+                "grouping": "deployment",
+                "original_grouping": "deployment",
+                "config_constraints": {
+                  "type": "string"
+                }
+              }
+            },
+            {
+              "key": "vsi_access_tags",
+              "custom_config": {
+                "grouping": "deployment",
+                "original_grouping": "deployment",
+                "config_constraints": {
+                  "type": "string"
+                }
+              }
+            },
+            {
+              "key": "auto_generate_ssh_key"
+            },
+            {
+              "key": "ssh_public_keys"
+            },
+            {
+              "key": "existing_ssh_key_ids",
+              "custom_config": {
+                "type": "vpc_ssh_key",
+                "grouping": "deployment",
+                "original_grouping": "deployment",
+                "config_constraints": {
+                  "selection": "multi_select"
+                }
+              }
+            },
+            {
+              "key": "existing_secrets_manager_instance_crn"
+            },
+            {
+              "key": "existing_secrets_manager_endpoint_type"
+            },
+            {
+              "key": "ssh_key_secret_group_name"
+            },
+            {
+              "key": "ssh_key_secret_name"
+            },
+            {
+              "key": "machine_type"
```

Comment:
We will want this to show in the required tab since profiles are different per region